values.yaml

```diff
@@ -12,61 +12,109 @@
   install: false
-metricsServer:
+metrics-server:
+  # install: install the metrics server, this should be true for the first installation into the cluster as it resides in kube-system to provide a single cluster-wide metrics server
   install: false
 dashboard:
+  # install: install routing through the ingress controller to a pre-installed kubernetes dashboard i.e. https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/
   install: false
 kafka:
+  # install: install kafka streaming platform
   install: true
+redis:
+  # install: install redis persistence and caching data storage platform
+  install: true
+redis-cluster:
+  # install: install clustered redis persistence and caching data storage platform
+  install: false
+
 namespaces:
   default: default
   dev: dev
   uat: uat
   prod: prod
+
 global:
+  # hostname: the external hostname of the ingress, this may be the cluster master or public load-balancer
   hostname: localhost
+
+  # nodes: the number of nodes in the cluster, this will be used to calculate the service scaling factors
   nodes: 1
+
+  # namespace: the target namespace, overridden by (1) --namespace (helm, the preferred override) or (2) global.uniqueNamespace=true, ingress path will include this name at the path root (default truncated to null)
   namespace: default
+
+  # uniqueNamespace: if true (and this is a new install) the generated deployment name will be used as the namespace name too, ingress path will include this name at the path root eg. localhost/lazy-goat/palisade
   uniqueNamespace: false
+
+  # repository: the name of the docker image repository that is the source of local application images that was populated during the project build process and will now be referenced in the pods
   repository: palisade
+
+  # see charts/kafka/charts/zookeeper
   zookeeperConnectString: palisade-zookeeper:2181
-  localMountKafka:
-    enabled: true
-  localMount:
-    enabled: true
-    volumeHandle: dummy
-  classpathjars:
-    directory: /tmp/Palisade/classpathjars
+
+  # persistent-volume and associated claims configurations
   persistence:
-    storageClass:
-    annotations: {}
-    accessMode:
-    - "ReadWriteOnce"
-    persistentVolumeReclaimPolicy: "Retain"
-    persistenceVolumeAccessModes:
-    - "ReadWriteOnce"
-    - "ReadOnlyMany"
-    enabled: true
-    hostpath: /tmp/Palisade
-    storageTierName:
-    - "classpath"
-    selectors:
-      classpath: "classpathjars"
-    size: 2Gi
-    pvcSizes:
-      policy:
-        size: 1Gi
+    # classpathJars: Store used by all services within palisade to dynamically load datatypes (Users, Resources, Policies/Rules, Serialisers)
+    classpathJars:
+      name: palisade-classpath-jars
+      # mountPath: internal mountpoint for classpath jars directory, see Dockerfile ENTRYPOINT -Dloader.path and PodSpec.containers.volumeMounts.mountPath
+      mountPath: "/usr/share/classpath-jars"
```

Comment:
This should just be "/usr/share/" as the rest is composed in the deployment.yaml so that the classpath directory is below the executable jar target.